app/Notifications/UserAccessBranchNotification.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Notifications;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Notification;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+
+class UserAccessBranchNotification extends Notification
+{
+    use Queueable;
+
+    protected $user;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @return void
+     */
+    public function __construct($user, $branches)
+    {
+        $this->user = $user;
+        $this->branches = $branches;
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return ['mail'];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage)
+                    ->greeting('Hello! ' . $this->user->name)
+                    ->subject($this->user->email . ' has access to the branch.')
```

Comment:
Change the notification text to be:
{{ branch }} has granted access to your account ({{ email }})